CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
-We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
-
 ## Instructions
 
 These steps will guide you through contributing to this project:
 
 - Fork the repo
 - Clone it and install dependencies
 
-		git clone https://github.com/YOUR-USERNAME/typescript-library-starter
+		git clone https://github.com/YOUR-USERNAME/alma-js-client
```

Comment:
`git clone https://github.com/alma/alma-js-client` ?

Reply:
Right! The template I had used assumed you had first forked/copied over their files to another repo!